ios/brave-ios/Sources/Brave/Frontend/Browser/Favicons/LargeFaviconView.swift

```diff
@@ -14,8 +14,56 @@ struct FaviconUX {
   static let faviconBorderWidth = 1.0 / UIScreen.main.scale
 }
 
+struct FaviconConfiguration {
+  let faviconBorderColor: UIColor
+  let faviconBorderWidth: CGFloat
+
+  static let defaultConfig = FaviconConfiguration(
+    faviconBorderColor: FaviconUX.faviconBorderColor,
+    faviconBorderWidth: FaviconUX.faviconBorderWidth
+  )
+}
+
 /// Displays a large favicon given some favorite
 class LargeFaviconView: UIView {
+  let config: FaviconConfiguration?
+
+  init(
+    config: FaviconConfiguration? = nil
+  ) {
+    self.config = config
+    super.init(frame: .zero)
+
+    layer.cornerRadius = 8
+    layer.cornerCurve = .continuous
+    if let config = self.config {
+      layer.borderColor = config.faviconBorderColor.cgColor
+      layer.borderWidth = config.faviconBorderWidth
```

Comment:
Doesn't seem like you need to store the configuration atm, as you never use it outside of init